Sources/Fluent/QueryBuilder/QueryBuilder+Aggregate.swift

```diff
@@ -71,13 +63,14 @@ extension QueryBuilder {
     ///
     /// - parameters:
     ///     - method: Aggregate method to use.
-    ///     - field: Field to find max for.
+    ///     - field: Field to find aggregate for.
     ///     - type: `Decodable` type to decode the aggregate value as.
+    ///     - default: Optional default to use.
     /// - returns: A `Future` containing the aggregate.
-    public func aggregate<D, T>(_ method: Database.QueryAggregate, field: KeyPath<Result, T>, as type: D.Type = D.self) -> Future<D>
-        where D: Decodable
+    public func aggregate<D, T>(_ method: Database.QueryAggregate, field: KeyPath<Result, T>, as type: D.Type = D.self, default: D?) -> Future<D>
```

Comment:
Moving from `Decodable` to `Codable` is a breaking change as well. I can't see how this would affect anyone, but we need to consider this.

Reply:
Did you mean from `Codable` to `Decodable`?

Reply:
This is now resolved. I'm using `.literal` instead of `.encodable` to avoid having to make the value `Encodable`. Using the `String(describing:)` initializer feels a little clumsy but it does pass the tests.